Cineaste/ViewController/SearchMovies/SearchMoviesViewController+SwipeHint.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  SearchMoviesViewController+SwipeHint.swift
+//  Cineaste App
+//
+//  Created by Wolfgang Timme on 7/14/19.
+//  Copyright © 2019 spacepandas.de. All rights reserved.
+//
+
+import UIKit
+
+extension SearchMoviesViewController {
+    /// Shows a hint for the swipe actions of the table view cells.
+    func animateSwipeActionHint() {
+        guard let cell = tableView.visibleCells.first as? SearchMoviesCell else { return }
+
+        guard shouldAnimateShowSwipeActionHint else { return }
+
+        cell.animateSwipeHint()
+
+        rememberThatSwipeActionHintWasAnimated()
+    }
+
+    /// MARK: Persistency
+
+    private var swipeActionHintShowCountKey: String {
```

Comment:
We already write and read some values to/from UserDefaults. Could you have a look a the implementation of `PromtForReviewPersistence` or `UsernamePersistence` and add another one for something like `SwipeActionHintPersistence`? Maybe these enums are not a good idea anymore, as there are three already, but I really like to stay consistent for this.

Reply:
Yes, consistency is definitely desirable. 👍🏼 I've added an `enum` accordingly.